pwndbg/symbol.py

```diff
@@ -87,12 +79,10 @@ def autofetch():
     if pwndbg.gdblib.android.is_android():
         return
 
-    if not remote_files_dir:
-        remote_files_dir = tempfile.mkdtemp()
+    remote_files_dir = pwndbg.file.remote_files_dir()
+    if remote_files_dir not in get_directory().split(":"):
```

Comment:
isn't this logic different?

Reply:
With the old logic, `symbol.py` could check if the global variable was `None` to determine whether it should add it to `debug-file-directory`. But now that the global is in `file.py`, to do the same thing I'd need some other global variable to check. Instead, I just check if we've already added it to `debug-file-directory`, and if not, then we add it. If you have other suggestions for how to rewrite this, let me know.

Reply:
one thing to consider: what if the directory has ':' in it?

Reply:
or maybe it doesnt matter much because we use tempfiles?

Reply:
Right, we don't need to worry about any weird directory names because we use tempfiles.